pkgs/os-specific/linux/batman-adv/default.nix

```diff
@@ -16,6 +16,10 @@ stdenv.mkDerivation rec {
     sha256 = cfg.sha256.${pname};
   };
 
+  patches = [
+    ./fix-skb_vlan_eth_hdr-conflict-in-stable.patch
```

Comment:
use `fetchpatch2` instead with https://git.open-mesh.org/batman-adv.git/commitdiff_plain/be69e50e8c249ced085d41ddd308016c1c692174?hp=74d3c5e1c682a9efe31b75e8986668081a4b5341.

Reply:
how do i get the correct sha256sum?

Reply:
TOFU, build it and you'll see it.